rest_api/packaging/rest_api.toml.example

```diff
@@ -0,0 +1,15 @@
+#
+# Sawtooth Lake -- REST API Configuration
+#
+
+# The port for the api to run on
+#   port = 8080
+
+# The host for the api to run on
+#   host = "127.0.0.1"
+
+# The url to connect to a running Validator
+#   stream_url = "tcp://localhost:40000"
```

Comment:
We should standardize on what we call this across all components (TPs, rest api, etc.) connecting to the component endpoint.